packaging/debian/debian/spectrum2.service

```diff
@@ -1,13 +1,21 @@
 [Unit]
-Description=spectrum2
+Description=An open source instant messaging transport
+Documentation=http://spectrum.im/documentation/
 After=network.target
 
 [Service]
 Type=forking
+
+User=spectrum2
```

Comment:
As proposed by others, I propose to drop this line for the sake of an easier setup and leave dropping the privileges to spectrum_manager.

Reply:
What about using systemd-sysusers?